### PR TITLE
:sparkles: Updated to podman 2.0.3 json format

### PR DESCRIPTION
**Pull request by rmarting**
podman 2.0.3 json format includes a set of differences in the schema (e.g: Names as an array, State instead of Status, ...). 

This commit includes some changes to update them.

With these minimal changes the gnome extension works successfully again with podman 2.0.3. Some extra staff should be done to  manage:

* Show `Created` attribute with some human readable format.
* Show `Ports` array in a human readable format.

Some changes are described in the following issues in podman:

* https://github.com/containers/podman/issues/6980
* https://github.com/containers/podman/issues/6594

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,21 +14,23 @@ See the [gnome extensions page](https://extensions.gnome.org/extension/1500/cont
 
 # Install from source
 
-First pick the right branch by the gnome-shell-X.XX version: 
+First pick the right branch by the `gnome-shell-X.XX` version: 
 
 | branch | gnome-shell version |
 | --- | --- |
 | master | 3.36.1 |
+| 3.36   | 3.36.4 |
```
Comment (rgolangh) [inline README.md:22]: The podman version is something that needs handling as well. I'm thinking on adding per podman release json parsing handling. Some people may newer version of gnome but old podman versions and vice versa.

**Comment by rgolangh** [comment]
@rmarting thank yo so much for this. This is not the first time podman breaks. Hopefully this is 
going to be more stable (Id instead of ID, State/Status and so on)

Important note about indentation - its deliberately like that and taken from many source code examples in the gnome-shell repo.

File: extension.js [hunk 2]
```diff
@@ -156,132 +159,141 @@ var PopupMenuItem = GObject.registerClass(
...
-        this.menu.addMenuItem(new PopupMenuItem("Created", container.Created));
-        this.menu.addMenuItem(new PopupMenuItem("Ports", container.Ports));
+    class extends PopupMenu.PopupSubMenuMenuItem {
+        _init(container, name) {
+            super._init(container.Names[0]);
+            this.menu.addMenuItem(new PopupMenuItem("Status", container.State));
+            this.menu.addMenuItem(new PopupMenuItem("Id", container.Id));
+            this.menu.addMenuItem(new PopupMenuItem("Image", container.Image));
+            this.menu.addMenuItem(new PopupMenuItem("Command", container.Command));
+            this.menu.addMenuItem(new PopupMenuItem("Created", timeConverter(container.Created)));
```
Comment (rgolangh) [inline extension.js:238]: We can actually just use `CreatedAt`

**Comment by rgolangh** [comment]
I'm going to close this in favour of something bit more generic #15 - thanks a lot for this PR!
Hopefully it would help to keep up with future changes and still leave the extension in tact.

**Comment by rmarting** [comment]
Cool Roy! Your approach is more clean than my initial proposal. I like it!
It was a pleassure to contribute into your gnome-extensión (my first time at all) :laughing:

**Pull request closed without merging**